storage/google/cloud/storage/blob.py

```diff
@@ -73,6 +75,9 @@ class Blob(_PropertyMixin):
     _CHUNK_SIZE_MULTIPLE = 256 * 1024
     """Number (256 KB, in bytes) that must divide the chunk size."""
 
+    _STORAGE_CLASSES = (
+        'STANDARD', 'NEARLINE', 'MULTI_REGIONAL', 'REGIONAL', 'COLDLINE')
```

Comment:
Don't we already have this [**somewhere**][1]?

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/2c608f95d7d5db2cbfdce77770f523a489b513f4/storage/google/cloud/storage/bucket.py#L93-L94